documentation/plain_rules.md

```diff
@@ -190,6 +191,197 @@ First time an update action using trust token is triggered, Perseo interacts wit
+    'value': '${varValue}'
+}
+```
+If `varValue` value is for example `"Good morning"`, this attribute will take `"Good morning"` as value.
+
+If `varValue` value is for example `1234`, this attribute will take `"1234"` as value.
+
+- Literal Text:
+```json
+{
+    'name':'textLiteral',
+    'type': 'Text',
+    'value': 'Hello world'
+}
+```
+This attribute will take `"Hello world"` as value.
+
+- Literal Number:
+```json
+{
+    'name':'textNumberLiteral',
+    'type': 'Text',
+    'value': 67.8
+}
+```
+This attribute will take `"67.8"` as value.
+
+- Literal Boolean:
+```json
+{
+    'name':'textBoolLiteral',
+    'type': 'Text',
+    'value': true
+}
+```
+This attribute will take `"true"` as value.
+
+With `DateTime` type attributes, Perseo will try to parse the value to DateTime format.
+
+Date as String:
+```json
+{
+    'name':'dateString',
+    'type': 'DateTime',
+    'value': '2018-12-05T11:31:39.00Z'
+}
+```
+This attribute will take `"2018-12-05T11:31:39.000Z"` as value.
+
+Date as Number in milliseconds:
+```json
+{
+    'name':'dateString',
+    'type': 'DateTime',
+    'value': 1548843229832
+}
+```
+This attribute will take `"2019-01-30T10:13:49.832Z"` as value.
+
+Date from variable.
+```json
+{
+    'name':'dateString',
+    'type': 'DateTime',
+    'value': '${dateVar}'
+}
+```
+If `dateVar` value is for example `1548843229832` (as Number or String), this attribute will take `"2019-01-30T10:13:49.832Z"` as value.
+
+You can use the `__ts` field of a Perseo DateTime attribute to fill a DateTime attribute value without using any `cast()`. For example, if the var are defined as follow in the rule text, `ev.timestamp__ts? as dateVar`, `dateVar` will be a String with the Date in milliseconds, for example `"1548843060657"` and Perseo will parse this String with to a valid DateTime as `2019-01-30T10:11:00.657Z`.
+
+With `None` type attributes, Perseo will set the value to `null` in all cases.
+
+None Attribute:
+```json
+{
+    'name':'nullAttribute',
+    'type': 'None',
+    'value': 'It does not matter what you put here'
+}
+```
+This attribute will take `null` as value.
+
+```json
+{
+    'name':'nullAttribute2',
+    'type': 'None',
+    'value': null
+}
+```
+This attribute will take `null` as value.
+
+
+**Complete example using NGSv2 update action in a rule:**
+
+```json
+{
+   "name":"blood_rule_update",
+   "text":"select *,\"blood_rule_update\" as ruleName, *, ev.BloodPressure? as Pressure from pattern [every ev=iotEvent(BloodPressure? > 1.5 and type=\"BloodMeter\")]",
+   "action":{
+      "type":"update",
+      "parameters":{
+         "id":"${id}_example",
+         "version": 2,
+         "attributes": [
+             {
+                 "name":"pressure",
+                 "type":"number",
+                 "value": "${Pressure}"
+             }
+         ]
+      }
+   }
+}
+```
+
+Note that using NGSIv2 the BloodPressure attribute is a Number and therefore it is not necessary to use `cast()`.
```

Comment:
I like this! :)

Current rules do an overuse `cast()` and NGSIv2 will allow to simplify a lot these EPL framents.

NTC (just a comment)